sdk/core/azure-core/src/main/java/com/azure/core/util/tracing/Tracer.java

```diff
@@ -57,6 +57,11 @@
      */
     String SPAN_BUILDER_KEY = "builder";
 
+    /**
+     * Key for {@link Context} which indicates that the context contains the Azure resource provider namespace.
+     */
+    String AZ_NAMESPACE_KEY = "az.namespace";
```

Comment:
Can we include `tracing` in the name of this context key? This will make it clear as to what this key is used for. AZ_NAMESPACE is too generic.

Reply:
but wouldn't coming from `Tracer` class help to understand that it is coming from tracing concepts.
I think it would repetitive to mention it in the variable name and class name too.

Reply:
No, context is a property bag that is passed around quite a lot in all layers of our library. Having a key named `az.namespace` in context will not necessarily be clear as to what this namespace is and why it's used.

Reply:
I think there is a slight mismatch in communication here. Srikanta is talking about the string "az.namespace", not the variable name "AZ_NAMESPACE_KEY" (despite the fact he used "AZ_NAMESPACE" in his comment). His argument is valid - imagine have a Map or a Context full of keys, one of which is "az.namespace" - it isn't clear where that came from or how it is relevant. If it had a more specific value then users might better infer that it is tracing related.

Reply:
Updated both the key and value to include tracing, i.e `AZ_TRACING_NAMESPACE_KEY` and `az.tracing.namespace`. The span attribute will still have the key `az.namespace`